### DISCUSSION TITLE
Possible "bug" report" - Share/Shared spelling error

### DISCUSSION THREAD

**Discussion by RJC0235** [streetcomplete/StreetComplete#5758] [Feedback]
Hello, This is the first time I have ever commented on a Github, so please be patient if i have done anything wrong, Github is like seeing angels for the first time and my mind scrambles whenever trying to navigate it.

I noticed recently there could be a possible spelling mistake when Streetcomplete asks if a sidewalk has a segregated / shared cycleway on the pavement / sidewalk.

I noticed this recently, so it may have already been fixed, but street complete likes to add "cycleway=both" + "cycleway:both=**share**_sidewalk" when i believe it should be "shared" with a D at the end. My thoughts on this are is its used with a d on the end everywhere else for when a Road shares a cycle-lane. e.g. [https://wiki.openstreetmap.org/wiki/Tag:cycleway%3Dshared_lane](https://wiki.openstreetmap.org/wiki/Tag:cycleway%3Dshared_lane)

However, the Wiki also doesn't seem to have anything where it defines if it should be "share" or "shared" when shared with a sidewalk

Many thanks,
0235

**Comment by FloEdelmann** [comment]
Can you share a changeset where StreetComplete added this tag? When searching for the value in the code, it's only listed once as "treat as imprecise and worth replacing", so StreetComplete (at least currently) doesn't tag it.

**Reply by matkoniecz** [reply]
I can imagine scenario when SC mapper split way with this tag present already

**Reply by RJC0235** [reply]
I have had a look through the history, and it seems that between me updating some information via street complete, and now, someone else came and manually added "share_sidewalk" instead of shared. False alarm.

i feel quite silly actually not checking how it was edited in the version history. seems to be a false alarm from me, and not helped that a search here also came up with a few discussions about it 🤦‍♂️
